components/Landing/Banner.js

```diff
@@ -0,0 +1,62 @@
+import { Container, Grid, Typography, Button, Hidden } from '@material-ui/core';
+import ArrowDownwardIcon from '@material-ui/icons/ArrowDownward';
+
+const Banner = () => {
```

Comment:
Is this the banner? Maybe a rename like Hero would be better? I just think a banner would be better suited to a text and picture strip component. if not then I think banner is ok
![image](https://user-images.githubusercontent.com/68363001/121826252-c94dca80-cc84-11eb-9f28-0d2b4a9ad3f9.png)

Reply:
Thanks. I actually forgot that it was named Hero. tyty